### PR TITLE
[8.0] [ADD] module database_cleanup

### PR DESCRIPTION
**Pull request by anthony-muschang**
This pull request adds a module to clean up after a migration.

Original proposal on Launchpad:
https://code.launchpad.net/~therp-nl/openupgrade-tools/7.0-add-database_cleanup/+merge/203633

The original authors are @StefanRijnhart and @guewen. The migration to 8.0 has been made by @sbidoul.

### CODE REVIEW

File: database_cleanup/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    'category': 'Tools',
+    'data': [
+        'view/purge_modules.xml',
+        'view/purge_models.xml',
+        'view/purge_columns.xml',
+        'view/purge_tables.xml',
+        'view/purge_data.xml',
+        'view/menu.xml',
+        ],
+    'description': """\
```
Comment (pedrobaeza) [inline database_cleanup/__openerp__.py]: Put description in README.rst file

**Comment by pedrobaeza** [comment]
The rest LGTM.

**Comment by sbidoul** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**